src/Bicep.Core/Emit/ExpressionConverter.cs

```diff
@@ -233,8 +233,10 @@ private LanguageExpression ConvertAccessExpression(AccessExpression expression,
         {
             var (@base, properties, safeAccess) = ConvertBaseExpression(expression);
 
-            if (expression is ModuleOutputPropertyAccessExpression)
+            if (expression is ModuleOutputPropertyAccessExpression moduleOutputExpression)
```

Comment:
Could we add a boolean flag (e.g. `IsSecure` to `ModuleOutputPropertyAccessExpression`, so that we don't have to introduce the name check here?

Reply:
Will try this, thanks!